kratos/solving_strategies/strategies/solving_strategy.h

```diff
@@ -356,11 +349,19 @@ class SolvingStrategy
 
     /**
      * @brief Operations to get the pointer to the model
-     * @return mpModelPart: The model part member variable
+     * @return *mpModelPart: The model part member variable
      */
-    inline ModelPart& GetModelPart()
+    ModelPart& GetModelPart()
+    {
+        return *mpModelPart;
+    };
+
+    /**
+     * @brief Operations to get the pointer to the model
+     * @return *mpModelPart: The model part member variable
+     */
+    const ModelPart& GetModelPart() const
     {
-        KRATOS_ERROR_IF_NOT(mpModelPart) << "ModelPart in the SolvingStrategy is not initialized" << std::endl;
```

Comment:
I think it is important to leave this error!

Reply:
Done.